mega-linter-runner/lib/runner.js

```diff
@@ -134,9 +133,6 @@ ERROR: Docker engine has not been found on your system.
         commandArgs.push(...["-e", envVarEqualsValue]);
       }
     }
-    if (commandArgs.containername) {
-      commandArgs.push(...["--name", commandArgs.containername]);
```

Comment:
This is most likely a matter of my own ignorance, but I am not seeing why the previous order of options would work any better or worse than the new order from reading the [`docker run`](https://docs.docker.com/engine/reference/commandline/run/) documentation. Would you mind adding a brief explanation of why this change fixes the bug to the commit message?

Reply:
To be honest, I tried to find the relevant thing in docs, but couldn't. However, if I move the `--name` argument after the `-v` it no longer works :(

Reply:
Sounds like the root cause is Docker’s documentation then. Thank you for educating me.